### DISCUSSION TITLE
Visualizing single or multi peak accessibility on UMAP embeddings

### DISCUSSION THREAD

**Discussion by ARFarghli** [GreenleafLab/ArchR#1947] [Feature Requests]
Hi there, thank you for developing an excellent tool with a wonderfully accessible manual!

Describe the problem that your feature request would address
I am detailing various distal TREs in my cancer of interest. I want to visualize how frequently a single or group of TREs are accessible within cells by looking at a UMAP. 

Describe the solution you'd like
An easy method to visualize peak accessibility of a single TRE or multiple TREs. For the latter, a coloring scheme that reflects the number of TREs are are accessible within each cell.

**Comment by rcorces** [comment]
https://github.com/GreenleafLab/ArchR/blob/c61b0645d1482f80dcc24e25fbd915128c1b2500/R/ModuleScore.R#L1-L21
